PluginOracleNet/API/Discover/GetAllRelatedEntities.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Aunalytics.Sdk.Plugins;
+using Google.Protobuf.Collections;
+using PluginOracleNet.API.Factory;
+
+namespace PluginOracleNet.API.Discover
+{
+    public static partial class Discover
+    {
+        private const string ColSourceTableSchema = "SOURCE_TABLE_SCHEMA";
+        private const string ColSourceTableName = "SOURCE_TABLE_NAME";
+        private const string ColSourceColumn = "SOURCE_COLUMN";
+        private const string ColForeignTableSchema = "FOREIGN_TABLE_SCHEMA";
+        private const string ColForeignTableName = "FOREIGN_TABLE_NAME";
+        private const string ColForeignColumn = "FOREIGN_COLUMN";
+        private const string ColRelationshipName = "SOURCE_CONSTRAINT";
+
+        private const string GetForeignKeysQuery = @"SELECT a.OWNER          SOURCE_TABLE_SCHEMA,
```

Comment:
align all the column names instead of having one up on the first row out of alignment